packages/docs/content/foundation/css-normalization.mdx

```diff
@@ -1,19 +1,21 @@
 ---
-title: Introducing a CSS reset
+title: CSS normalization
 order: 10
 ---
 
-import marginCollapse from '../../src/images/margin-collapse.png'
+import { Alert } from '@cmsgov/design-system';
 
-The design system's CSS reset is a collection of HTML element-specific styles that simplifies the development process while providing cross-browser sensible defaults to build from.
+The design system includes a set of global CSS rules that aim to normalize (remove the differences in) browser defaults. By making CSS normalization part of the design system, teams don't need to bring their own, and the design system CSS doesn't need to include extra defensive CSS to anticipate differences in normalization code across different products. Our global normalization rules do not completely reset element styles but instead set elements' base styles to sensible defaults, following current best practices.
 
 Our reset CSS is bundled with the rest of the design system's styles, so to use it all you need to do is [import the design system's CSS](/getting-started/for-developers/#including-the-css) using your preferred method. Even if you haven't imported one of our components, your application should visually align with the design system.
 
-If you're curious to see what changes have been made, you can preview our reset styles in [Storybook](/storybook/?path=/story/reset-overview--overview). We'll break down the major changes for you to review below.
+<Alert heading="Why do we need to normalize CSS?">
+Each browser has default styles that it applies to various HTML elements. Not all browser have the same default styles (called user-agent styles), which means that websites without custom CSS will look different when viewed from different browsers. Even when creating custom CSS, web developers must take those different default styles into account and make sure they are overridden appropriately. This can be tedious and error-prone, so a common practice among developers is to either remove or normalize (remove the differences in) those user-agent styles with a collection of base CSS rules before applying any component CSS that controls the look-and-feel of the site.
```

Comment:
Typo:
- "Not all browser have the same" should be "Not all browsers have the same"

Reply:
Thanks!